src/dll/edbgEcmdDll.C

```diff
@@ -1617,9 +1619,9 @@ uint32_t dllPutMemProc(const ecmdChipTarget & i_target, uint64_t i_address, uint
       blockSize = 8;
     }
 
-    rc = adu_putmem_io(adu_target, i_address, buf, i_bytes, blockSize);
+    rc = mem_read(mem_target, i_address, buf, i_bytes, blockSize, true);
```

Comment:
I have not looked at the API - but shouldn't this be `mem_write` and not `mem_read` for a put?

Reply:
oops sorry.. copy paste mistake